Feat: organisms(molecules 단위) 생성 [FRONT-169]

sidsidsidsids opened the pull request:
### Changes
organisms 생성 (구성 atoms에 대한 수정 있음)
 - EmojiBox
 - ExitBox
 - NicknameInputBox
 - PageChangeButton

<br>

### Test Checklist ☑️
- [x] 고려하지 않은 누락된 점이 있는지 확인해주세요
- [x] 기획에 맞게 구성되었는지 확인해주세요

### Screenshot(option)
![FRONT-169](https://github.com/BbeumbungE/Frontend/assets/122246882/bf48c361-dafd-4325-ad90-620c805390f5)

<br>

sidsidsidsids committed:
Feat: 이모지 박스 (게시글 상세 조회 이모지) 구현 [FRONT-169]
- 이모지 사이 공백 위해 Emoji.tsx 자체 margin-right 추가
- 세 이모지를 담은 div 형태로 구현
Related to: issue #169

sidsidsidsids committed:
Feat: 닉네임 입력 박스 구현 [FRONT-169]
- TextInputBox 코드 구조 수정
- InputButton 코드 구조 수정
- 두 atom을 담은 div로 형성
Related to: issue #169

sidsidsidsids committed:
Feat: 나가기 버튼 박스 구현 [FRONT-169]
- 화살표 버튼과 나가기 텍스트를 담은 div로 구현
Related to: issue #169

sidsidsidsids committed:
Feat: 페이지 좌우 버튼 구현 [FRONT-169]
- 두 버튼을 담고 포지션이 absolute인 div로 구현
Related to: issue #169

sidsidsidsids committed:
Feat: Emoji onClick props 추가 [FRONT-169]
- Emoji 각각 마다 다른 onClick 받도록 설정
Related to: issue #169

sidsidsidsids committed:
Feat: 페이지 전환 버튼 공백 공간 클릭 불가 해소 [FRONT-169]
- 기존 div로 감쌌을 때 화살표 가운데 공백에 있는 요소 클릭 불가
- 전체 div 클릭 이벤트 비활성화, 버튼 부분만 활성화함으로 해소
Related to: issue #169

YeryunJung commented:
pointer-events로 클릭을 해결하신 부분을 보고 새로운 속성을 공부해볼 수 있었습니다👍👍👍
구두로 말씀드렸던 것처럼 z-index를 빼고 pointer-events로 대체하는 방향으로 변경해주시면 감사하겠습니다!
더불어서 placeholder도 일단은 삭제 부탁드립니다!

sidsidsidsids committed:
Feat: 일부 organisms 프로젝트 구성에 맞게 수정 [FRONT-169]
- NicknameInputBox z-index 제거
- PageChangeButton z-index 제거
Related to: issue #169
